{{cookiecutter.project_slug}}/LICENSE.md

```diff
@@ -649,14 +649,14 @@ Also add information on how to contact you by electronic and paper mail.
 If the program does terminal interaction, make it output a short notice like
 this when it starts in an interactive mode:
 
-    <program>  Copyright (C) <year>  <name of author>
+    <program>  Copyright © <year>  <name of author>
     This program comes with ABSOLUTELY NO WARRANTY; for details type 'show w'.
     This is free software, and you are welcome to redistribute it
     under certain conditions; type 'show c' for details.
 
 The hypothetical commands `show w` and `show c` should show the appropriate
 parts of the General Public License. Of course, your program's commands might be
-different; for a GUI interface, you would use an “about box”.
+different; for a GUI, you would use an “about box”.
```

Comment:
Yeah this, for example, is a bad change. The GPL says you cant change the wording of the license.